test/benchmarks/Benchmarks.Trace/Benchmarks.Trace.csproj

```diff
@@ -28,6 +28,7 @@
 
   <ItemGroup>
     <PackageReference Include="BenchmarkDotNet" Version="0.12.1" />
+    <PackageReference Include="JetBrains.Profiler.Api" Version="1.1.7" />
```

Comment:
I've never used it before, but it looks like `JetBrains.Profiler.SelfApi` doesn't require other software (like dotTrace and dotMemory) to be installed on the machine. What do you think of using that instead?

Reply:
I didn't use that because I don't know if could cause a license problem.